Packs/Cisco-umbrella/Integrations/CiscoUmbrellaEnforcement/CiscoUmbrellaEnforcement.py

```diff
@@ -0,0 +1,197 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+''' IMPORTS '''
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+
+class Client(BaseClient):
+    """
+    Client to use in the Cisco Umbrella Enforcement integration. Overrides BaseClient
+    """
+
+    def __init__(self, base_url, verify, proxy, api_key):
+        super().__init__(base_url=base_url, verify=verify, proxy=proxy, ok_codes=(200, 202, 204))
+        self.api_key = api_key
+
+    def get_domains_list(self, page: str = '', limit: str = '') -> list:
```

Comment:
let's pass the limit as `int` so we will cast only once in `prepare_suffix` and not in every loop iteration
```suggestion
    def get_domains_list(self, page: str = '', limit: int = 50) -> list:
```

Reply:
I think that converting limit into an int before the loop will do the job.

Reply:
ok